packages/ses/index.d.ts

```diff
@@ -69,8 +69,14 @@ export interface RedirectStaticModuleInterface {
   importMeta?: any;
 }
 
+export interface AliasModuleInterface {
+  alias: string;
+  compartment: Compartment;
+}
+
 export type StaticModuleType =
   | RedirectStaticModuleInterface
+  | AliasModuleInterface
```

Comment:
would it be possible to use the Redirect type with slight modifications? Alias and Redirect seem conceptually close.
That would require a way to tell if it should go through `loadRecord` or `memoizedLoadWithErrorAnnotation` and potentially some simplification of `Redirect...`?   
I want to make sure that we're intentionally making sure the same module instance gets cached under both specifiers. (I'm not sure this is what other engines or bundlers are doing when things get aliased, they probably are)
A Redirect that says "look me up again in moduleMap under this other name before moving forward" is potentially a viable alternative. 

I'm not sure if that's a good idea, just trying to go through the consequences and play the devil's advocate or checking if they're intentional.

Reply:
I would also like to entertain the possibility that this could have the same shape as a redirect. The implementation is likely the same, but with separate handling for the cases where a redirect appears in the moduleMap/moduleMapHook or the importHook.

That is to say, if we rename `alias` to `specifier`, does this break? Does this require us to distinguish an alias/redirect record differently?